src/Element/TraversableElement.php

```diff
@@ -157,7 +157,11 @@ public function fillField($locator, $value)
             throw new ElementNotFoundException($this->getDriver(), 'form field', 'id|name|label|value|placeholder', $locator);
         }
 
-        $field->setValue($value);
+        if ($field->getTagName() === "select") {
```

Comment:
I've found https://github.com/minkphp/MinkSelenium2Driver/blob/master/src/Selenium2Driver.php#L612-L626 code for Selenium2Driver where this is already done. Not sure if other drivers do that as well.